test/api/v01/output_test.rb

```diff
@@ -74,6 +74,17 @@ def test_no_day_week_num
     refute_includes csv_data.first, 'day_week_num'
   end
 
+  def test_returned_skills
+    vrp = VRP.basic
+    vrp[:vehicles].first[:skills] = [['skill_to_output']]
+    vrp[:services].first[:skills] = ['skill_to_output']
+
+    response = post '/0.1/vrp/submit', { api_key: 'demo', vrp: vrp }.to_json, 'CONTENT_TYPE' => 'application/json'
+    result = JSON.parse(response.body)['solutions'].first
+    activities = result['routes'].collect{ |r| r['activities'] }.flatten
```

Comment:
flat_map

Reply:
can you check all collect{}.flatten's you added and replace them with flat_map?

I don't understand why rubocop doesn't bug you with this :thinking: